### PR TITLE
Makefile: sync with apiserver

### PR DESCRIPTION
**Pull request by sttts**
Makefiles had diverged.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,24 +27,24 @@ endif
...
+	imagebuilder -f Dockerfile -t $(IMAGE_REPOSITORY_NAME)/origin-cluster-kube-controller-manager-operator .
 .PHONY: images
 
 clean:
 	$(RM) ./cluster-kube-controller-manager-operator
 .PHONY: clean
 
 , := ,
-IMAGES ?= cluster-kube-controller-manager-operator
+IMAGES ?= cluster-kube-apiserver-operator
```
Comment (mfojtik) [inline Makefile]: typo

File: Makefile [hunk 2]
```diff
@@ -27,24 +27,24 @@ endif
...
+	imagebuilder -file hack/lib/Dockerfile-origin-release --build-arg "IMAGE_REPOSITORY_NAME=$(IMAGE_REPOSITORY_NAME)" --build-arg "IMAGES=$(QUOTED_IMAGES)" -t "$(IMAGE_REPOSITORY_NAME)/origin-release:latest" hack
 	@echo
 	@echo "To install:"
 	@echo
-	@echo "  DOCKER_ORG=$(DOCKER_ORG) make images"
-	@echo "  docker push $(DOCKER_ORG)/origin-cluster-kube-controller-manager-operator"
-	@echo "  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=docker.io/$(DOCKER_ORG)/origin-release:latest bin/openshift-install cluster --log-level=debug"
+	@echo "  IMAGE_REPOSITORY_NAME=$(IMAGE_REPOSITORY_NAME) make images"
+	@echo "  docker push $(IMAGE_REPOSITORY_NAME)/origin-release:latest"
+	@echo "  docker push $(IMAGE_REPOSITORY_NAME)/origin-cluster-kube-apiserver-operator"
```
Comment (mfojtik) [inline Makefile]: copy&paste

File: Makefile (hunk 1 above)
Comment (sttts) [reply Makefile]: blä

Comment (sttts) [reply Makefile]: fixed

File: Makefile (hunk 2 above)
Comment (sttts) [reply Makefile]: fixed

**Comment by tnozicka** [comment]
sync 
/lgtm

We are going to have a share place for build machinery soon to fix this. Also likely the release stuff is about to get replaced as I have learned some tricks from Seth ;)

**Pull request merged**